docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[search-aggregations-bucket-frequent-items-aggregation]]
+=== Frequent items aggregation
+++++
+<titleabbrev>Frequent items</titleabbrev>
+++++
+
+A bucket aggregation which finds frequent item sets. Finding frequent item sets 
+is a form of association rules mining that identifies items that are often occur 
+together. Items that are frequently purchased together or log events that tend 
+to co-occur are examples of frequent item sets. Finding frequent item sets helps 
+to discover relationships between different data points (items) in a data set 
+and to create association rules based on those relationships.
+
+
+==== Syntax
+
+A `frequent_items` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+"frequent_items": {
+  "minimum_set_size": 3,
+  "fields": [
+    {"field": "my_field_1"},
+    {"field": "my_field_2"}
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+.`frequent_items` Parameters
+|===
+|Parameter Name |Description |Required |Default Value
+|`field` |(array) Fields to analyze. | Required |
+|`minimum_set_size` | (integer) The minimum size of one item set. | Optional | `1`
+|`minimum_support` | (integer) The minimum support of one item set. | Optional | `0.1`
```

Comment:
I read this and thought "I don't know what minimum support means", so it might be nice to link to the details here
```suggestion
|`minimum_support` | (integer) The <<frequent-items-minimum-support,minimum support>> of one item set. | Optional | `0.1`
```